src/daemons/GraphDaemon.cpp

```diff
@@ -130,26 +133,35 @@ int main(int argc, char *argv[]) {
     gServer->setThreadStackSizeMB(5);
 
     // Setup the signal handlers
-    setupSignalHandler();
+    status = setupSignalHandler();
+    if (!status.ok()) {
+        LOG(ERROR) << status;
```

Comment:
Please append more log info, just display status message maybe confuse.

Reply:
the status is return from SignalHandler::installInternal, and it will take enough info to get the error reason. thx.